lib/parser/parse-options.js

```diff
@@ -10,7 +10,7 @@ function parseOptions(input, callback, keyValueDelim, groupDelim) {
     if (kv.length !== 2) {
       continue;
     }
-    var k = kv[0];
+    var k = kv[0].trim();
     var v = kv[1];
```

Comment:
can we add a trim here too, just in case?

Reply:
Thanks for taking a look! Pushed up that change here.